code/modules/cargo/packs.dm

```diff
@@ -2182,6 +2217,24 @@
 	crate_name = "art supply crate"
 	crate_type = /obj/structure/closet/crate/wooden
 
+/datum/supply_pack/misc/captain_pen
+	name = "Captain Pen"
+	desc = "A spare Captaun fountain pen."
+	access = ACCESS_CAPTAIN
+	cost = 10000
+	contains = list(/obj/item/pen/fountain/captain)
+	crate_name = "captain pen"
+	crate_type = /obj/structure/closet/crate/secure/weapon //It is a combat pen
+
+/datum/supply_pack/misc/captain_headset
```

Comment:
Ditto for what Poojawa said here. Why is this necessary when traitors and lings are already more than capable of getting all-access radio keys on their own without help from cargo.

Reply:
Also borgs and standing caps, people tend to take the keys and cryo.